Controller/ResourceController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\Controller;
+
+use Symfony\Cmf\Component\Resource\RepositoryRegistryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Hateoas\HateoasBuilder;
+use Symfony\Component\HttpFoundation\Response;
+use Hateoas\UrlGenerator\UrlGeneratorInterface;
+use JMS\Serializer\SerializerBuilder;
+use JMS\Serializer\SerializerInterface;
```

Comment:
can we decouple from JMSSerializer and also allow the Symfony Serializer? (as it's going to be awesome in Symfony 2.7 and already quite is in 2.6)